apps/logging/tests/test_loggers_management_command.py

```diff
@@ -1,20 +1,37 @@
+import copy
```

Comment:
START-CODE-FOR-REVIEW-SECTION

This adds tests to validate metric results that are logged.

There are 4x sections of testing. See the comment labels for each section for info about what is being tested:  TEST #1 - 4